engines/grim/detection.cpp

```diff
@@ -56,6 +56,15 @@ static const ADExtraGuiOptionsMap gameGuiOptions[] = {
 			false
 		}
 	},
+	{
+		GAMEOPTION_LOAD_DATAUSR,
```

Comment:
Mmh, then maybe it's not the right way to go. That option is applicable to the whole engine rather than per datafile variant. I was a bit surprised to find the existing checkbox in detection.cpp file, but did not think too much about it.